README.md

```diff
@@ -1,30 +1,29 @@
-# StoryBook Boilerplate
-
-Simple boilerplate to kickstart your styleguide endeavors with [StoryBook](https://storybook.js.org)
+# Bonline Component Library
 
 Includes:
 
-- Jest Tests configured with coverage
-- Prettier
-- Babel Config
-- Sample config for storybook
+- Storybook
+- Testing suite
+- Rollup
+- Components and base styles
 
-## Install
+## Testing Changes in Projects
 
-```bash
-git clone git@github.com:SaraVieira/storybook-boilerplate.git
-cd storybook-boilerplate
-rm -rf .git
-yarn
-yarn start
-```
+### tl;dr
+
+1. Checkout your Fork of this Repo
+2. Run this in the fork root folder: `yarn link`
+3. Run this in the root folder of the project you would like to use this module: `yarn link b-lib`
 
-## Update to your info
+You will now be able to import this module like any normal npm module like so:
 
-```bash
-git remote add origin git@github.com:YourUsername/YourRepo.git
 ```
+import { SimplePopup } from 'b-lib';
+```
+
+## When Done with Dev on your Project
 
-Also make sure to update the package.json with your info
+1. install module like so: `yarn add https://github.com/youruser/b-lib#yourtag --force`
```

Comment:
we wont be using personal tags, only branch names

tags will be pushed to master on release

Reply:
@robguy21 can you amend the Deploy process info hre?